src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -68,6 +69,9 @@ class JenkinsJobManager {
             if (startOnCreate && missingJob.jobName.contains("Rev.com-featurebranch-build-feature")) {
```

Comment:
We could... but... while I was doing this task I read about https://jenkins.io/doc/book/pipeline/ and wondered if we'd just pursue that approach after the Jenkins upgrade ticket.